packages/oui-navbar/src/link/navbar-link.controller.js

```diff
@@ -0,0 +1,20 @@
+export default class {
+    constructor ($element, $timeout) {
+        "ngInject";
+
+        this.$element = $element;
+        this.$timeout = $timeout;
+    }
+
+    $postLink () {
+        this.$timeout(() =>
+            this.$element
+                .addClass("oui-navbar-list__item")
+        );
+    }
+
+    // Return value of "ui-sref"
+    getFullSref () {
+        return `${this.state}(${JSON.stringify(this.stateParams)})`;
```

Comment:
I don't get this piece of code, isn't `this.stateParams` supposed to be a `String` ? If it is, maybe you could throw an error if it isn't a `String`

Reply:
No, stateParams is an object